src/midi/midi_fluidsynth.cpp

```diff
@@ -856,15 +856,23 @@ std::string format_sf2_line(size_t width, const std_fs::path& sf2_path)
 
 MIDI_RC MidiHandlerFluidsynth::ListAll(Program* caller)
 {
+	// Find SoundFont from user config. FluidSynth may not be Open so it
+	// must be done here.
 	auto* section = static_cast<Section_prop*>(control->GetSection("fluidsynth"));
 	const auto sf_spec = parse_soundfont_pref(section->Get_string("soundfont"));
-	const auto sf_name      = std::get<std::string>(sf_spec);
+	const std_fs::path found_soundfont = find_sf_file(
+	        std::get<std::string>(sf_spec));
+
 	const size_t term_width = INT10_GetTextColumns();
 
-	auto write_line = [caller](bool do_highlight, const std::string& line) {
+	auto write_line = [caller, term_width, this](const std_fs::path& sf2_path) {
```

Comment:
Just use `[&]` because you've listed `this` anyway?

Reply:
Sure.  I always forget the syntax for lambdas.  Changed as you suggested.